include/swupd.h

```diff
@@ -199,6 +224,18 @@ extern int get_value_from_path(char **contents, const char *path, bool is_abs_pa
 extern int get_version_from_path(const char *abs_path);
 extern timelist init_timelist(void);
 
+static inline int bsearch_helper(const void *A, const void *B)
+{
+	return strcmp(*(const char **)A, ((struct filerecord *)B)->filename);
+}
```

Comment:
This function is not used anywhere aside from extra_files.c, so probably does not need to live in this header.